.prospector.yml

```diff
@@ -0,0 +1,82 @@
+output-format: grouped
+
+# strictness: veryhigh
+# doc-warnings: no
+# test-warnings: no
+max-line-length: 120
+# pep8: full
+
+ignore-paths:
+  - __pycache__/
+  - .github/
+  - venv/
+
+mypy:
+  run: true
+  options:
+    ignore-missing-imports: true
+    follow-imports: skip
+
+pylint:
+  options:
+    extension-pkg-allow-list: mypy
+  disable:
+    - too-few-public-methods
+    - missing-docstring
+    - django-not-available
+    - logging-fstring-interpolation
+    - logging-format-interpolation
+    - N803
+    - bare-except
+    - cyclic-import
+    - no-else-return
+    - too-many-locals
+    - simplifiable-if-expression
+    - import-error
+    - useless-object-inheritance
+    - protected-access
+    - unused-import
+    - unused-variable
+    - pointless-string-statement
+    - inconsistent-return-statements
+    - broad-exception-caught
+    - singleton-comparison
+    - invalid-name
+    - line-too-long
+    - unused-argument
+    - redefined-builtin
+    - too-many-function-args
```

Comment:
We need to enable the disabled violation types and make the necessary modifications in the codebase to address them.